docs/src/main/sphinx/admin/dynamic-filtering.rst

```diff
@@ -3,35 +3,44 @@ Dynamic filtering
 =================
 
 Dynamic filtering optimizations significantly improve the performance of queries
-with selective joins by avoiding reading of data that would be filtered by join condition.
+with selective joins by avoiding reading of data that would be filtered by join
+condition.
 
-Consider the following query which captures a common pattern of a fact table ``store_sales``
-joined with a filtered dimension table ``date_dim``:
+Consider the following query which captures a common pattern of a fact table
+``store_sales`` joined with a filtered dimension table ``date_dim``:
 
     SELECT count(*)
     FROM store_sales
     JOIN date_dim ON store_sales.ss_sold_date_sk = date_dim.d_date_sk
     WHERE d_following_holiday='Y' AND d_year = 2000;
 
-Without dynamic filtering, Trino pushes predicates for the dimension table to the
-table scan on ``date_dim``, and it scans all the data in the fact table since there
-are no filters on ``store_sales`` in the query. The join operator ends up throwing away
-most of the probe-side rows as the join criteria is highly selective.
+Without dynamic filtering, Trino pushes predicates for the dimension table to
+the table scan on ``date_dim``, and it scans all the data in the fact table
+since there are no filters on ``store_sales`` in the query. The join operator
+ends up throwing away most of the probe-side rows as the join criteria is highly
+selective.
 
-When dynamic filtering is enabled, Trino collects candidate values for join condition
-from the processed dimension table on the right side of join. In the case of broadcast joins,
-the runtime predicates generated from this collection are pushed into the local table scan
-on the left side of the join running on the same worker.
+When dynamic filtering is enabled, Trino collects candidate values for join
+condition from the processed dimension table on the right side of join. In the
```

Comment:
right side of the join

(Adding "the". Compare with "left side of the join" later on.